docs/tools/vdb_table/data/singlestoredb.json

```diff
@@ -0,0 +1,191 @@
+    "source_url": "https://docs.singlestore.com/cloud/developer-resources/functional-extensions/working-with-vector-data/",
+    "comment": ""
+  },
+  "facets": {
+    "support": "full",
+    "source_url": "",
+    "comment": "Supported through SQL aggregations"
+  },
+  "geo_search": {
+    "support": "full",
+    "source_url": "https://docs.singlestore.com/cloud/reference/sql-reference/data-types/geospatial-types/",
+    "comment": ""
+  },
+  "multi_vec": {
+    "support": "full",
+    "source_url": "",
+    "comment": "Multiple vector columns can be created and queried"
+  },
+  "sparse_vectors": {
+    "support": "none",
+    "source_url": "",
+    "comment": ""
+  },
+  "bm25": {
+    "support": "full",
+    "source_url": "https://docs.singlestore.com/cloud/reference/sql-reference/full-text-search-functions/bm-25/",
+    "comment": ""
+  },
+  "full_text": {
+    "support": "full",
+    "source_url": "https://docs.singlestore.com/cloud/developer-resources/functional-extensions/working-with-full-text-search/",
+    "comment": ""
+  },
+  "embeddings_text": {
+    "support": "none",
+    "source_url": "",
+    "comment": ""
+  },
+  "embeddings_image": {
+    "support": "none",
+    "source_url": "",
+    "comment": ""
+  },
+  "embeddings_structured": {
+    "support": "none",
+    "source_url": "",
+    "comment": ""
+  },
+  "rag": {
+    "support": "none",
+    "source_url": "",
+    "comment": ""
+  },
+  "recsys": {
+    "support": "none",
+    "source_url": "",
+    "comment": ""
+  },
+  "langchain": {
+    "support": "full",
+    "source_url": "https://python.langchain.com/docs/integrations/vectorstores/singlestoredb",
+    "comment": ""
+  },
+  "llamaindex": {
+    "support": "partial",
+    "source_url": "https://docs.llamaindex.ai/en/stable/api_reference/readers/singlestore/",
+    "comment": ""
+  },
+  "managed_cloud": {
+    "support": "full",
+    "source_url": "https://www.singlestore.com/product-overview/",
+    "comment": ""
+  },
+  "pricing": {
+    "value": "",
+    "source_url": "https://www.singlestore.com/pricing/",
+    "comment": ""
+  },
+  "in_process": {
+    "support": "none",
+    "source_url": "",
+    "comment": ""
+  },
+  "multi_tenancy": {
+    "support": "full",
+    "source_url": "",
+    "comment": ""
+  },
+  "disk_index": {
+    "support": "full",
+    "source_url": "",
+    "comment": ""
+  },
+  "ephemeral": {
+    "support": "none",
+    "source_url": "",
+    "comment": ""
+  },
+  "sharding": {
+    "support": "full",
+    "source_url": "https://docs.singlestore.com/cloud/getting-started-with-singlestore-helios/about-singlestore-helios/sharding/",
+    "comment": ""
+  },
+  "doc_size": {
+    "bytes": null,
+    "unlimited": true,
+    "source_url": "",
+    "comment": ""
+  },
+  "vector_dims": {
+    "value": 10000,
+    "unlimited": false,
+    "source_url": "",
+    "comment": ""
+  },
+  "index_types": {
+    "value": [
+      "DOT_PRODUCT",
+      "EUCLIDEAN",
+      "COSINE"
```

Comment:
These are distance metrics. This column is for supported index types like HNSW and IVF. Please check files for other DBs and revise